app/src/test/java/org/simple/clinic/newentry/PatientEntryScreenControllerTest.kt

```diff
@@ -223,6 +229,39 @@ class PatientEntryScreenControllerTest {
     verify(screen, times(4)).showEmptyStateError(true)
   }
 
+  @Test
+  fun `when input validation errors are emitted, they must be sent to analytics`() {
```

Comment:
I like to think that the test names should be from a business POV and not include any technical implementation details like "emitted", etc. Do you think we can change this?